pkgs/development/libraries/SDL/default.nix

```diff
@@ -25,13 +25,14 @@ stdenv.mkDerivation rec {
 
   # Since `libpulse*.la' contain `-lgdbm', PulseAudio must be propagated.
   propagatedNativeBuildInputs =
```

Comment:
@aszlig (fbc307a183): why should all these be native? They seem (mostly) libs to be linked against.